SimpleCalculator.py

```diff
@@ -76,12 +76,18 @@ def find_max_value(values):
 
 
 # TODO this function should take list of values, then return average of those values
+list_of_values = [1,2,3,4]
+
 def calculate_average(values):
     average = 0
+    for average in values:
```

Comment:
Ok, this one is major problem, you shouldn't be calculating it like that, result is good, but there is useless code, that takes space in file, and it also takes longer to run. Python sum() function returns sum of list that was passed to it, so if you use it you don't need to iterate through the list. The way your code is behaving is that you interate through input list, and then you write to element of list average value of this list.  Code  sum(values) / len(values) already calculates average value so you don't need to interate once again. You could also calculate by yourself sum of elements by interating:
sum = 0
for value in values:
    sum = sum + value  # you add value to sum and write it to sum once again / this way you can increase value od 
                                     #  sum by value 
then you will calculate average as:
average = sum / len(values)

You could also calculate by yourself len of values, not only sum:

sum = 0
lenght = 0
for value in values:
    sum = sum + value
    lenght = lenght + 1 # increase length by 1 with each iteration

then average will look:

average = sum / length

But I think that solution with Python sum() and len() is nice :) So you should only remove unnecessary for loop.

I hope that this comment will be helpful, if not we can make some small session and we will try to write every iteration on papaer so you will better understand